Gordon360/Controllers/NewsController.cs

```diff
@@ -22,10 +22,12 @@ namespace Gordon360.Controllers
     public class NewsController : GordonControllerBase
     {
         private readonly INewsService _newsService;
+        private readonly IWebHostEnvironment _webHostEnvironment;
 
         public NewsController(CCTContext context, MyGordonContext myGordonContext, IWebHostEnvironment webHostEnvironment, ServerUtils serverUtils)
         {
             _newsService = new NewsService(myGordonContext, context, webHostEnvironment, serverUtils);
+            _webHostEnvironment = webHostEnvironment;
```

Comment:
As I mentioned in my other comments, we don't the `webHostEnvironment` because we don't actually need to load the image and send it as a base64 string, we can just send the URL to the frontend.

That means you can remove the `webHostEnvironment` dependency.

Also, instead of taking all the parameters for an `INewsService`, we can just take an `INewsService` param directly and allow dependency injection to instantiate it with the correct parameters.

```suggestion

        public NewsController(INewsService newsService)
        {
            _newsService = newsService;
```